pom.xml

```diff
@@ -231,6 +231,21 @@
               </descriptors>
             </configuration>
           </execution>
+          <execution>
+            <id>app-tgz</id>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+            <configuration>
+              <appendAssemblyId>true</appendAssemblyId>
+              <attach>true</attach>
+              <tarLongFileMode>posix</tarLongFileMode>
+              <descriptors>
+                <descripto>src/main/assembly/bin.xml</descripto>
```

Comment:
descriptor

Reply:
Hmm, I think that tag label must be arbitrary then, given the build succeeded and attached the artifact.  Maven?!?